packages/react-composites/tests/browser/call/hermetic/fixture.ts

```diff
@@ -98,7 +98,7 @@ export function defaultMockCallAdapterState(participants?: MockRemoteParticipant
 /**
  * Create the default {@link MockCallAdapterState}for call readiness hermetic e2e tests.
  */
-export function defaultMockCallAdapterStateCallReadiness(
+export function defaultMockCallAdapterStateDeviceDisabled(
```

Comment:
Consider calling `defaultMockCallAdapter` state and then modifying the devices instead of recreating the whole state here. 

Imagine that we add one more field to the state, and need to provide a default value for that. We want to be able to set that in *1* place for the tests instead of for each of the `defaul*State` helpers.